src/style.css

```diff
@@ -1 +1,35 @@
 /* add styles here */
+:root{
+  --main-images: (./images/star-active.svg)
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(./images/star.svg);
+  background-position: center;
+  margin-right: 4px;
```

Comment:
Consider using a consistent unit for margins. If you're using `px` for widths and heights, you might want to use `px` for margins as well for consistency, unless there's a specific reason for using `4px` here.